src/dags/gob_dags.py

```diff
@@ -0,0 +1,72 @@
+"""
```

Comment:
I think we don't need __init__.py in dags

Reply:
For testing we do.

In GOB testing is an essential part of the architecture. No code should be checked in without tests. And in order to write tests efficiently you should refer to python packages. It is more readable test code.